### ISSUE TITLE
When updated to the latest version 0.91.0 ClosedXml has a hard reference to FastMember.Signed

### ISSUE THREAD

**Issue by alexey-aurea** [ClosedXML/ClosedXML#691]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

e.g. 0.91.0

**What is the current behavior?**

ClosedXml has a hardcoded reference to the FastMember.Signed v 1.1.0. When FastMember.Signed is updated to 1.2.0 the code throws following exception
System.IO.FileLoadException : Could not load file or assembly 'FastMember.Signed, Version=1.0.0.9, Culture=neutral, PublicKeyToken=9e8f22703bef9a29' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at ClosedXML.Excel.XLTheme..ctor()

**What is the expected behavior or new feature?**

not to throw the exception and allow any versions of FastMember.Signed packed to be used which is higher or equal to required

**Did this work in previous versions of our tool?  Which versions?**

No

## Reproducibility
**This is an important section. Read it carefully. Failure to do so will cause a 'RTFM' comment.**

create new project and reference ClosedXml 0.91.0
in the main do this:
public static void Main()
{
    var workbook = new XLWorkbook();
}
run the code - it works.

Update FastMember.Signed package to the latest 1.2.0
Run the code again - you will see the exception

**Code to reproduce problem:**
```c#
public static void Main()
{
    var workbook = new XLWorkbook();
}

```
- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Comment by vbjay** [comment]
Add an assembly redirect to your app.config. Raise another issue or even
pull request to update the nuget package.

**Comment by alexey-aurea** [comment]
but there should not be a hard reference to a particular nuget package

**Comment by Artur-A** [comment]
Facing the same problem with the latest version, invalid reference to the old FastMember.Signed instead of  v.1.2.0.

**Comment by vbjay** [comment]
I see. So yank the reference and replace with the nuget package and send in
a pull request.

**Comment by igitur** [comment]
Did the FastMember public key change?

**Comment by igitur** [comment]
See https://github.com/mgravell/fast-member/issues/50

**Comment by RafVandesande** [comment]
Could you reconsider the wontfix label?

Should be a simple fix. The FileVersion attribute in 1.2.0 is '0.0.0.0' (not AssemblyVersion). This causes binding redirects to be generated like so:
```xml
<dependentAssembly>
  <assemblyIdentity name="FastMember.Signed" publicKeyToken="9e8f22703bef9a29" culture="neutral"/>
  <bindingRedirect oldVersion="0.0.0.0-0.0.0.0" newVersion="0.0.0.0"/>
</dependentAssembly>
```
Note the range on the **oldVersion** attribute.
So, you won't be getting anything unless you're asking for version '0.0.0.0'

A manual workaround could be:
```xml
<dependentAssembly>
  <assemblyIdentity name="FastMember.Signed" publicKeyToken="9e8f22703bef9a29" culture="neutral"/>
  <bindingRedirect oldVersion="0.0.0.0-1.2.0.0" newVersion="0.0.0.0"/>
</dependentAssembly>
```

**Comment by igitur** [comment]
I'm not sure where you suggest those changes go. But if it's so simple, please submit a PR.

**Comment by RafVandesande** [comment]
Oops, sorry, this should go to the FastMember project, not ClosedXml. Nothing you can (or should) do about it here. My bad...

**Comment by ArgeKumandan** [comment]
this update screwed up my production environment :( :( :(

**Comment by igitur** [comment]
You should have tested in a non-production environment first.

**Comment by igitur** [comment]
This not a ClosedXML issue. Refer all comments to https://github.com/mgravell/fast-member/issues/50

**Comment by igitur** [comment]
FastMember fixed this in v1.3.0
